### PR TITLE
Improve Home page

### PR DESCRIPTION
**Pull request by e-aakash**
Displays all form created by user in /home
image

### CODE REVIEW

**Comment by pulsejet** [comment]
Pretty!

**Comment by e-aakash** [comment]
Updated home view. Can create new form by clicking the new button
Forms displayed in collapsable tabs

Screenshot from 2019-07-16 21-56-57

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+      <mat-expansion-panel *ngFor="let item of forms">
+        <mat-expansion-panel-header>
+          {{item.Name}}
+        </mat-expansion-panel-header>
+    
+        <a mat-raised-button color="primary" href="{{origin}}/edit/{{item.ID}}" >Edit</a>    
+        <a mat-raised-button color="primary" href="{{origin}}/m/{{item.ID}}" style="margin-left: 20px">Form Link</a>
+        <a mat-raised-button color="primary" href="{{origin}}/response/{{item.ID}}-{{item.Token}}" 
+          style="margin-left: 20px">Responses</a>
```
Comment (pulsejet) [inline src/app/home/home.component.html]: Inline CSS 😒

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
+      }
+    })
   }
 
+  removeSection(id: number, name: string) {
+    if (confirm('Delete form ' + name + '? This action is irreversible! \nAll responses will also be deleted.')) {
+      const observable = this.http.delete(`/api/form/${id}`);
+
+      observable.subscribe((result: any) => {
+        alert('Form ' + name + ' has been deleted!')
```
Comment (pulsejet) [inline src/app/home/home.component.ts]: ES6 string interpolation please!

**Pull request merged**